SevenRedLibrary/Service.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System.Collections.Generic;
+
+namespace SevenRedLibrary
+{
+    public static class Service
+    {
+        /// <summary>
+        /// Find Won deck
+        /// </summary>
+        /// <param name="WhiteDeck"></param>
+        /// <param name="BlackDeck"></param>
+        /// <returns>String Who won</returns>
+        public static string FindWinner(List<Card> WhiteDeck, List<Card> BlackDeck)
```

Comment:
Create a class named `CardsCombination` that will store info about a set of cards, methods to find higher card and validation if we can add new card to the set (remember that we cannot add duplicate cards)